clientv3/balancer.go

```diff
@@ -162,6 +163,18 @@ func (b *simpleBalancer) Up(addr grpc.Address) func(error) {
 
 func (b *simpleBalancer) Get(ctx context.Context, opts grpc.BalancerGetOptions) (grpc.Address, func(), error) {
 	var addr string
+
+	if !opts.BlockingWait {
+		b.mu.RLock()
+		addr = b.pinAddr
+		upEps := len(b.upEps)
+		b.mu.RUnlock()
+		if upEps == 0 {
+			return grpc.Address{Addr: ""}, nil, errors.New("there is no address available")
```

Comment:
anyway, defined `NoAddressAvailable` in `balancer.go`. Let me know there's  a better place to define errors. I'll relocate the error.